src/raven/processors/sanitize_data.cr

```diff
@@ -62,10 +71,15 @@ module Raven
       JSON.parse_raw(string) rescue nil
```

Comment:
I'm not sure if it's the same result, but looks like `parse_raw` was removed in `0.25.0`, but you can possibly replace with [this](https://github.com/crystal-lang/crystal/pull/5183/files#diff-95be945dbfb4392613e72d811ebbddf8R78) `JSON.parse(string).raw)`

Reply:
indeed, I tried to wrap up #28, but `parse.raw` now returns hash/array of `JSON::Any` here but after further changes I'm still getting compiler errors related to casting 😅  (not fully familiar with introduced changes in 0.25 yet so I kinda have no idea what I'm doing 🐴 )

Reply:
Yep, probably [any_hash](https://github.com/Sija/any_hash.cr) shard should be updated accordingly.